lib/syntax.ml

```diff
@@ -107,10 +107,17 @@ let exists_grouping columns =
   List.exists (function Expr (e,_) -> is_grouping e | All | AllOf _ -> false) columns
 
 (* all columns from tables, without duplicates *)
-(* FIXME check type of duplicates *)
```

Comment:
this fixme disappeared, i think it is important, at some point we want to warn if there are columns with same name but different types

Reply:
Okay, I'll return it. I just thought my PR put a dot to the ambiguity check, and this is why I deleted it.